### PR TITLE
Include Collaboration Features while generating the changelog

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Other: Include Collaboration Features packages in the changelog.

### CODE REVIEW

File: scripts/release/changelog.js [hunk 1]
```diff
@@ -14,16 +14,22 @@ const fs = require( 'fs' );
...
+	throw new Error( `The script assumes that the directory "${ CKEDITOR5_INTERNAL_PATH }" exists.` );
+}
+
 Promise.resolve()
 	.then( () => devEnv.generateChangelogForMonoRepository( {
 		cwd: process.cwd(),
 		packages: 'packages',
-		releaseBranch: 'release',
+		// TODO: Restore the "release" branch.
+		releaseBranch: 'i/ckeditor5-internal/652',
```
Comment (pomek) [inline scripts/release/changelog.js:32]:
```suggestion
		releaseBranch: 'release',
```

File: scripts/release/changelog.js [hunk 2]
```diff
@@ -41,7 +47,16 @@ Promise.resolve()
 			{
 				cwd: CKEDITOR5_INTERNAL_PATH,
 				packages: 'packages',
-				skipLinks: true
+				skipLinks: true,
+				// TODO: Remove the single line below.
+				releaseBranch: 'master'
```
Comment (pomek) [inline scripts/release/changelog.js:52]:
```suggestion
				skipLinks: true
```

File: scripts/release/changelog.js [hunk 3]
```diff
@@ -41,7 +47,16 @@ Promise.resolve()
...
+			},
+			{
+				cwd: COLLABORATION_FEATURES_PATH,
+				packages: 'packages',
+				skipLinks: true,
+				// TODO: Remove the single line below.
+				releaseBranch: 'master'
```
Comment (pomek) [inline scripts/release/changelog.js:59]:
```suggestion
				skipLinks: true
```

**Review by pomek — COMMENTED** [review]
After adjusting the CF repository, please, commit suggested changes.

File: scripts/release/changelog.js [hunk 4]
```diff
@@ -14,16 +14,22 @@ const fs = require( 'fs' );
...
+
 Promise.resolve()
 	.then( () => devEnv.generateChangelogForMonoRepository( {
 		cwd: process.cwd(),
 		packages: 'packages',
-		releaseBranch: 'release',
+		// TODO: Restore the "release" branch.
+		releaseBranch: 'i/ckeditor5-internal/652',
 		highlightsPlaceholder: true,
 		collaborationFeatures: true,
```
Comment (pomek) [inline scripts/release/changelog.js:34]: I guess we don't need this:

```
The CKEditor 5 Collaboration features changelog can be found here: https://ckeditor.com/collaboration/changelog.
```

in the changelog anymore. Let's remove it.

```suggestion
```

**Comment by ma2ciek** [comment]
All works fine and looks good to me. Once you check the changes in CF I'll merge it with the mentioned changes.

File: scripts/release/changelog.js [hunk 5]
```diff
@@ -14,18 +14,22 @@ const fs = require( 'fs' );
...
 
 const CKEDITOR5_INTERNAL_PATH = path.resolve( __dirname, '..', '..', 'external', 'ckeditor5-internal' );
+const COLLABORATION_FEATURES_PATH = path.resolve( __dirname, '..', '..', 'external', 'collaboration-features' );
 
 if ( !fs.existsSync( CKEDITOR5_INTERNAL_PATH ) ) {
 	throw new Error( `The script assumes that the directory "${ CKEDITOR5_INTERNAL_PATH }" exists.` );
 }
 
+if ( !fs.existsSync( COLLABORATION_FEATURES_PATH ) ) {
+	throw new Error( `The script assumes that the directory "${ CKEDITOR5_INTERNAL_PATH }" exists.` );
```
Comment (pomek) [inline scripts/release/changelog.js:24]:
```suggestion
	throw new Error( `The script assumes that the directory "${ COLLABORATION_FEATURES_PATH }" exists.` );
```

**Pull request merged**